app/src/main/java/com/example/taras/gascalculation/AppUI/MainActivity.java

```diff
@@ -0,0 +1,45 @@
+package com.example.taras.gascalculation.AppUI;
```

Comment:
Package name must be in lower case 
http://www.oracle.com/technetwork/java/codeconventions-135099.html

And so it is clear that everything in the package gascalculation apply to the application. Rename the package to ui